pkg/ccl/changefeedccl/protected_timestamps_test.go

```diff
@@ -523,7 +570,18 @@ func TestPTSRecordProtectsTargetsAndSystemTables(t *testing.T) {
 	// Change the user's password to update the users table.
 	sqlDB.Exec(t, `ALTER USER test WITH PASSWORD 'testpass'`)
 
-	time.Sleep(2 * time.Second)
```

Comment:
I think for this test to be valid , we likely want to sleep some amount of time to ensure that `ts` is beyond the gc threshold before we try to gc the range.